ubcpi/serialize.py

```diff
@@ -0,0 +1,248 @@
+IMAGE_ATTRIBUTES = {'position': 'image_position', 'show_image_fields': 'show_image_fields', 'alt': 'image_alt'}
+
+
+class UpdateFromXmlError(Exception):
+    """
+    Error occurred while deserializing the OpenAssessment XBlock content from XML.
+    """
+    pass
+
+
+class ValidationError(UpdateFromXmlError):
+    """
+    The XML definition is not semantically valid.
+    """
+    pass
+
+
+def _safe_get_text(element):
+    """
+    Retrieve the text from the element, safely handling empty elements.
+
+    Args:
+        element (lxml.etree.Element): The XML element.
+
+    Returns:
+        unicode
+    """
+    return unicode(element.text) if element.text is not None else u""
```

Comment:
@nedbat strange. I have tests in non-ASCII characters (Chinese characters: https://github.com/ubc/ubcpi/blob/master/ubcpi/test/data/parse_question_xml.json#L30). It doesn't have any problem. However, if I change it to unicode(x, "utf-8"), the test gave me this:

```
decoding Unicode is not supported
```

It seems element.text is already returning unicode.